pkg/util/os/os.go

```diff
@@ -43,3 +47,36 @@ func CurrentOS() OS {
 	}
 	panic("Unexpected OS type")
 }
+
+func CurrentExecutable() (string, error) {
+	currentExec, err := osext.Executable()
+	if err != nil {
+		return "", err
+	}
+	return currentExec, nil
+}
+
+// EnvironmentAsMap returns os.Environ as map.
+func EnvironmentAsMap() map[string]string {
```

Comment:
can we refactor this to a general method and split of a specified parameter? `func SplitStringAsMap()  map[string]string { }`